src/UI/class-settings-page.php

```diff
@@ -310,14 +310,15 @@ private function initialize_basic_section(): void {
 		);
 
 		// Get the API Key.
-		$h          = __( 'Your Site ID is your own site domain (e.g. `mydomain.com`).', 'wp-parsely' );
+		$h          = __( 'Your Site ID is your own site domain without <code>http(s)://</code> prefixes or trailing <code>/</code> (e.g. <code>mydomain.com</code>).', 'wp-parsely' );
```

Comment:
```suggestion
		$h          = __( 'Your Site ID is typically your own site domain without <code>http(s)://</code> prefixes or trailing <code>/</code> (e.g. <code>mydomain.com</code>).', 'wp-parsely' );
```

The string could actually be anything, so while the default is to use the domain, we can allow a little wiggle room by adding a "typically".